app/views/schools/manual_readings/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1><%= t('manage_school_menu.manage_manual_readings') %></h1>
+<% if @current_target.nil? %>
+  <p><%= t('schools.manual_readings.set_target_html', path: new_school_school_target_path(@school)) %></p>
+<% elsif @months.values.flatten.empty? %>
```

Comment:
Could we not link to the form if there's no need to use it? Saves adding menu items, etc that may not be needed?

Rather than checking what months are needed every time, then we could add a flag to the FuelConfiguration during regeneration to decide whether adding manual readings is worthwhile, then show the links?

It might be useful for internal admins to see previously supplied data though.